MovieJournalMVC/Model/MovieReview.swift

```diff
@@ -17,4 +17,8 @@ struct MovieReview: Equatable, Codable {
         self.title = title
         self.review = review
     }
+
+    static func ==(lhs: MovieReview, rhs: MovieReview) -> Bool {
```

Comment:
Since this is a struct, you should just implement `extension MovieReview: Equatable { }` and let the compiler synthesize the implementation for you, rather than doing it manually.

Reply:
👍  Added a MovieReview extension that implements Equatable.